js/localization.js

```diff
@@ -30,7 +30,7 @@ function tHTML(html, tag) {
     }
     const body = t.DOMParser.parseFromString(html, 'text/html').body;
     if (html.includes('i18n-')) {
-      tNodeList(body.getElementsByTagName('*'));
+      tNodeList($$('*', body));
```

Comment:
This code is executed during page load so it's better to keep getElementsByTagName as it's much-much-much faster than querySelectorAll used in $$.